2022-02-07 12:00:27,841 INFO    MainThread:23513 [wandb_setup.py:_flush():71] setting env: {}
2022-02-07 12:00:27,841 INFO    MainThread:23513 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-07 12:00:27,841 INFO    MainThread:23513 [wandb_init.py:_log_setup():371] Logging user logs to /home/sohui/code/DTC/code/wandb/run-20220207_120027-2okytfh0/logs/debug.log
2022-02-07 12:00:27,841 INFO    MainThread:23513 [wandb_init.py:_log_setup():372] Logging internal logs to /home/sohui/code/DTC/code/wandb/run-20220207_120027-2okytfh0/logs/debug-internal.log
2022-02-07 12:00:27,842 INFO    MainThread:23513 [wandb_init.py:init():404] calling init triggers
2022-02-07 12:00:27,842 INFO    MainThread:23513 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {'mdoel': 'DTC', 'server': 57, 'image size': '112,112,80'}
2022-02-07 12:00:27,843 INFO    MainThread:23513 [wandb_init.py:init():449] starting backend
2022-02-07 12:00:27,843 INFO    MainThread:23513 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-07 12:00:28,040 INFO    MainThread:23513 [backend.py:ensure_launched():199] starting backend process...
2022-02-07 12:00:28,779 INFO    MainThread:23513 [backend.py:ensure_launched():204] started backend process with pid: 23710
2022-02-07 12:00:28,783 INFO    MainThread:23513 [wandb_init.py:init():458] backend started and connected
2022-02-07 12:00:28,794 INFO    MainThread:23513 [wandb_init.py:init():519] updated telemetry
2022-02-07 12:00:28,799 INFO    MainThread:23513 [wandb_init.py:init():549] communicating current version
2022-02-07 12:00:30,472 INFO    MainThread:23513 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-07 12:00:30,473 INFO    MainThread:23513 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-02-07 12:00:30,781 INFO    MainThread:23513 [wandb_init.py:init():592] starting run threads in backend
2022-02-07 12:00:31,272 INFO    MainThread:23513 [wandb_run.py:_console_start():1816] atexit reg
2022-02-07 12:00:31,275 INFO    MainThread:23513 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2022-02-07 12:00:31,275 INFO    MainThread:23513 [wandb_run.py:_redirect():1695] Redirecting console.
2022-02-07 12:00:31,283 INFO    MainThread:23513 [wandb_run.py:_redirect():1751] Redirects installed.
2022-02-07 12:00:31,283 INFO    MainThread:23513 [wandb_init.py:init():619] run started, returning control to user process
2022-02-07 12:03:29,074 INFO    MainThread:23513 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 0
2022-02-07 12:03:29,076 INFO    MainThread:23513 [wandb_run.py:_restore():1758] restore
2022-02-07 12:03:31,695 INFO    MainThread:23513 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 576
  total_bytes: 576
}

2022-02-07 12:03:31,941 INFO    MainThread:23513 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 576
  total_bytes: 576
}

2022-02-07 12:03:32,222 INFO    MainThread:23513 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 576
  total_bytes: 576
}

